.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required
+dist: trusty
+before_install:
+  # Install LaTeX (for architecture diagram)
+  - sudo apt-get -qq update
+  - sudo apt-get install -y --no-install-recommends texlive-fonts-recommended texlive-latex-extra texlive-fonts-extra texlive-latex-recommended texlive-extra-utils latex-xcolor pgf
```

Comment:
Can't we use [the Travis APT Addon](https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-with-the-APT-Addon) to install dependencies declaratively instead of requiring sudo and doing it through a script?